features/_selinux/test/test_kernel_cmdline.py

```diff
@@ -0,0 +1,14 @@
+import pytest
+from helper.tests.file_content import file_content
+
+
+@pytest.mark.parametrize(
+    "file,args",
+    [
+        ("/etc/kernel/cmdline", "security=selinux"),
+    ]
+)
+
+
+def test_kernel_cmdline(client, file, args):
```

Comment:
please skip this test for githubActionRunner and firecracker feature by changing this to

```python
def test_kernel_cmdline(client, file, args, non_feature_githubActionRunner, non_feature_firecracker):
```

and adding a `non_feature_firecracker` pytest fixture in `tests/conftest.py`

```python
@pytest.fixture
def non_feature_firecracker(testconfig):
    features = testconfig.get("features", [])
    if "firecracker" in features:
        pytest.skip('test is not supported on firecracker')
```